### DISCUSSION TITLE
does your sync also mixes up order

### DISCUSSION THREAD

**Discussion by nuttijay** [floccusaddon/floccus#1091] [Q&A]
Hi,

when i sync between my devices, bookmark order is changed / mixed up...

Am I missing a config option ?

cheers
nj

**Comment by marcelklehr** [comment]
Hi @nuttijay!
This should not happen, but is not unheard of, I've been able to replicate some of these reports recently and published fixes along with v4.12, but you should already be running that version, so there might still be some bugs left in the code.
Can you produce a step-by-step guide on how to reproduce this?

**Reply by pinpontitit** [reply]
> Hi @nuttijay!
> This should not happen, but is not unheard of, I've been able to replicate some of these reports recently and published fixes along with v4.12, but you should already be running that version, so there might still be some bugs left in the code.
> Can you produce a step-by-step guide on how to reproduce this?

Happening for me also.
Its difficult for me to give you step by step because I am myself surprised when it happens.

Generally it isnt mixed up, and when occasionally it arrives I already forgot what I did.

And even if I remember, I do so many things with my bookmarks thats its difficult to understand which one of my actions caused it.

Fot example I am using Floccus extension in many browsers: Chrome, librewolf, regular firefox, firefox nightly, and also Android Floccus, so I sync all those but not everyday, sometimes only one of them or two or three,....

Also I work a lot with my Bookmarks, thats a way of sorting my own mind, so I regularly, everyday, add, delete, sync, then delete, then add, then sort, then create folder, then delete folder, then sync, then rename bookmark, then rename folder, etc.
All of that in different browsers and the android app, 

so you understand its hard for me to give you better than the following:

Today, it happened. One bookmark(at least, the only one I noticed maybe) was mixed up.
It happened like that:
Earlier today, I added a new bookmark on my browser, and put it in my toolbar.
Then Later I sorted this bookmark and put it in the fifth position of my toolbar.
I dont remember if I did sync before sorting or after sorting, and neither if I edited or renamed this bookmark after saving it.

The only thing I know is : tonight on Floccus Android I can see this bookmark in first position of the root folder in the app.

In my browser it was last position of root folder, then fifth position, but never at first position.

Maybe it has something to do with closing the browser too rapidly during sync, or syncing browser and Android sometimes almost at the same time ?

**Comment by pinpontitit** [comment]
It happened again today.
I think it happens when syncing changes between browser and the android app.

I created two new bookmarks on browser, so they were at the end of my bookmark toolbar.
Then I synced in Android, and in Android they were immediately placed at the beginning of the other bookmarks, (instead of the end)
Then when I synced in browser, it changed to beginning accordingly with Android.

So, I guess, the problem comes from the Android app processing of bookmarks downloaded from the browser bookmarks.

Sorry I write a lot, but I'm just trying to give the more details possible to help reproduce.
